### PR TITLE
add custom image repo for keadm join beta

### PR DESCRIPTION
**Pull request by TianTianBigWang**
Signed-off-by: TianTianBigWang <zilong.wang@daocloud.io>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/image/image.go [hunk 1]
```diff
@@ -51,8 +51,12 @@ var edgeSet = Set{
...
 
-func EdgeSet(version string) Set {
+func EdgeSet(imageRepository, version string) Set {
 	set := edgeSet.Current(version)
+	if imageRepository != "" {
+		arr := strings.SplitN(set[EdgeCore], "/", 2)
+		set[EdgeCore] = imageRepository + arr[1]
+	}
 	set[EdgeMQTT] = "eclipse-mosquitto:1.6.15"
 	set[EdgePause] = "kubeedge/pause:3.1"
```
Comment (zc2638) [inline pkg/image/image.go:61]: All images need to be converted

Comment (TianTianBigWang) [reply pkg/image/image.go]: change

**Review by fisherxu — COMMENTED** [review]
Is this also needed for init?

**Comment by TianTianBigWang** [comment]
> Is this also needed for init?

It looks like `image` and `tag` are used in `init-beta.go`, I can use a custom image repository for `init-beta` in another pr if needed

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**